backend/climateconnect_main/settings.py

```diff
@@ -0,0 +1,134 @@
+"""
+Django settings for climateconnect_main project.
+
+Generated by 'django-admin startproject' using Django 2.2.11.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.2/ref/settings/
+"""
+
+import os
+from dotenv import find_dotenv, load_dotenv
+load_dotenv(find_dotenv('.pyenv'))
```

Comment:
`.pyenv` feels like the wrong name for this file, to me. I think it'd be easy to get it confused with [the pyenv project](https://github.com/pyenv/pyenv). Why not use `.env`?

Reply:
We have `.env` file for our frontend project so I thought the file name should be different. I can rename it to `.backend_env` or something similar. What do you think?

Reply:
I think `.backend_env` is clearer—let's do that.

Reply:
Done